src/components/backend-ai-login.ts

```diff
@@ -316,6 +317,11 @@ export default class BackendAILogin extends BackendAIPage {
     } else {
       this.allow_project_resource_monitor = true;
     }
+    if (typeof config.general === "undefined" || typeof config.general.openPortToPublic === "undefined" || config.general.allowProjectResourceMonitor === '' || config.general.openPortToPublic == false) {
```

Comment:
config.general.allowProjectResourceMonitor === '' should be  config.general.openPortToPublic === ''

Reply:
Oops... Thanks. I pushed with typo fixed.